vortex-array/src/patches.rs

```diff
@@ -124,11 +124,6 @@ impl Patches {
         }
     }
 
-    // TODO(ngates): remove this...
-    pub fn into_parts(self) -> (usize, usize, ArrayRef, ArrayRef) {
```

Comment:
there's on place where we needed it where we had owned patches so this should go